### ISSUE TITLE
Media is not playing at all

### ISSUE THREAD

**Issue by azrael783** [esphome/media-players#6]
I flashed the configuration from this repo using the latest version of esphome running on a Raspberry Pi 4 and connected it to Home-Assistant 2022.06.07 (hassos in a VM). I tried to play a locally stored mp3 file and also a radio stream. Both did not work. HA switched shortly to playing and then instantly back to not playing. I got this error message: `processAudioHeaderData(): can't see content in audioHeaderData`. Both local file and radio stream are working if you use a different media player (Sonos or browser).

**Comment by balloob** [comment]
What device?

**Comment by azrael783** [comment]
I am using a m5 Echo Atom.

**Comment by ananyevgv** [comment]
My locally saved mp3 file is not working. Radio stream. On two  devices LiraT 4.3 & ESP 32 wemos D1 + PCM5102A

**Comment by ananyevgv** [comment]
My locally saved mp3 file is not working. The radio stream works well. On two devices LiraT 4.3 & ESP 32 wemos D1 + PCM5102A

**Comment by ananyevgv** [comment]
On Larat 4.3, when using platform: esp32_touch, it stops working

**Comment by ananyevgv** [comment]
On LoraT 4.3, when using platform: esp32_touch, it stops working

**Comment by ananyevgv** [comment]
On LiraT 4.3, when using platform: esp32_touch, it stops working

**Comment by azrael783** [comment]
I flashed the demo firmware "BT speaker" from the m5 docs website. It is working but I had to increase the volume to the maximum on my cell phone to hear something. After this test I reflashed the esphome firmware but it still doesn't play any media :(

**Comment by dmshimself** [comment]
I have the same issue with an m5stack-atom-echo, so any advice appreciated.  Other media players play audio fine, but the atom-echo is silent.  I don't see any errors displayed and in the HA media sidebar, the pause button is displayed, but no media data, unlike with other media devices where the logo of the album shows.   The LED works, the button works, but not the audio.  

I've enclosed the log file from the device in case it helps.  It shows the audio being sent.  There isn't anything obvious in the home assistant log file or the ESPhome ones, but I'm happy to rummage around if there is anything else someone might like to see.

I'm on the latest HA and I've enclosed a screen shot of that too.

[logs_m5-atom-echo-d67d78_logs.txt](https://github.com/esphome/media-players/files/9865057/logs_m5-atom-echo-d67d78_logs.txt)

![Screenshot_2022-10-26_13-57-54](https://user-images.githubusercontent.com/6901481/197910107-bf13102b-cfc7-4c52-b4fb-332c97834467.png)
